lib/linters/duplicate_property.js

```diff
@@ -21,8 +21,8 @@ module.exports = {
             if (properties.indexOf(property.content) !== -1) {
                 results.push({
                     message: util.format(self.message, property.content),
-                    column: property.start.column,
-                    line: property.start.line
+                    column: property.start && property.start.column,
```

Comment:
these values shouldn't be `undefined` under any circumstances. if property.start isn't defined, then that's what your PR is setting it to.